packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -79,9 +78,9 @@ export async function getPortalsTradeQuote(
     .times(100)
     .toNumber()
 
-  const slippageTolerancePercentageDecimal =
-    input.slippageTolerancePercentageDecimal ??
-    getDefaultSlippageDecimalPercentageForSwapper(SwapperName.Portals)
+  const maybeSlippageTolerancePercentageOverride = input.slippageTolerancePercentageDecimal
```

Comment:
suggestion: just starting to read this so it may make more sense as I finish reviewing it, but feels like the user-selection/default *is* the regular slippage, and upstream is the override.

Maybe something like `userSlippageTolerancePercentageDecimalOrDefault?`

Reply:
I can get on board with that, I've removed all "override" terminology in favour of "user" in https://github.com/shapeshift/web/pull/7654/commits/d65f3d5877f925e154a1aec358681b08b2bd04a8